File: ICSharpCode.Decompiler.Tests/TestCases/VBPretty/Issue2192.vb
Imports System
Imports System.Linq
Public Class Issue2192
	Public Shared Sub M()
		Dim words As String() = {"abc", "defgh", "ijklm"}
		Dim word As String = "test"
		Console.WriteLine(words.Count(Function(w) w.Length > word.Length))
	End Sub
End Class

File: ICSharpCode.Decompiler.Tests/TestCases/VBPretty/Issue2192.cs
using System;
using System.Linq;
using System.Runtime.CompilerServices;

public class Issue2192
{
	public static void M()
	{
		string[] source = new string[3] { "abc", "defgh", "ijklm" };
		string text = "test";
		Console.WriteLine(source.Count([SpecialName] (string w) => w.Length > text.Length));
	}
}